.gitlab/ornl-ci.yml

```diff
@@ -31,6 +31,11 @@ env-setup:
     matrix:
       - HOST: [ "ascent" ]
         SPACK_ENV_NAME: [ "ascent" ]
+      - HOST: [ "summit" ]
+        SPACK_ENV_NAME: [ "rhel8-ppc64le" ]
+        COMPILER: [ "gcc@10.2.1" ]
+        TARGET_ARCH_OPT: [ "target=ppc64le" ]
```

Comment:
it is not used below, thus for now it could be removed (or extend the following commands with these parameters)